nfabackend/src/main/java/com/msg/nfabackend/entities/nfaCatalog.java

```diff
@@ -1,17 +1,304 @@
+		private BpPropertyTemplateNoConditionDe de;
+		private BpPropertyTemplateNoConditionEn en;
+		/**
+		 * @return the de
+		 */
+		public BpPropertyTemplateNoConditionDe getDe() {
+			return de;
+		}
+		/**
+		 * @param de the de to set
+		 */
+		public void setDe(BpPropertyTemplateNoConditionDe de) {
+			this.de = de;
+		}
+		/**
+		 * @return the en
+		 */
+		public BpPropertyTemplateNoConditionEn getEn() {
+			return en;
+		}
+		/**
+		 * @param en the en to set
+		 */
+		public void setEn(BpPropertyTemplateNoConditionEn en) {
+			this.en = en;
+		}
+	}
+	
+	public static class BpPropertyTemplateNoConditionDe {
+
+	    private String bezeichnung;
+	    private String erklaerung;
+	    private String characteristic;
+	    private String property;
+	    private String modalVerb;
+	    private String qualifyingEx;
+	    private String valueInput;
+	    private String verb;
+	    
+		/**
+		 * @return the bezeichnung
+		 */
+		public String getBezeichnung() {
+			return bezeichnung;
+		}
+		/**
+		 * @param bezeichnung the bezeichnung to set
+		 */
+		public void setBezeichnung(String bezeichnung) {
+			this.bezeichnung = bezeichnung;
+		}
+		/**
+		 * @return the erklaerung
+		 */
+		public String getErklaerung() {
+			return erklaerung;
+		}
+		/**
+		 * @param erklaerung the erklaerung to set
+		 */
+		public void setErklaerung(String erklaerung) {
+			this.erklaerung = erklaerung;
+		}
+		
+		/**
+		 * @return the characteristic
+		 */
+		public String getCharacteristic() {
+			return characteristic;
+		}
+		/**
+		 * @param characteristic the characteristic to set
+		 */
+		public void setCharacteristic(String characteristic) {
+			this.characteristic = characteristic;
+		}
+		/**
+		 * @return the property
+		 */
+		public String getProperty() {
+			return property;
+		}
+		/**
+		 * @param property the property to set
+		 */
+		public void setProperty(String property) {
+			this.property = property;
+		}
+		/**
+		 * @return the modalVerb
+		 */
+		public String getModalVerb() {
+			return modalVerb;
+		}
+		/**
+		 * @param modalVerb the modalVerb to set
+		 */
+		public void setModalVerb(String modalVerb) {
+			this.modalVerb = modalVerb;
+		}
+		/**
+		 * @return the qualifyingEx
+		 */
+		public String getQualifyingEx() {
+			return qualifyingEx;
+		}
+		/**
+		 * @param qualifyingEx the qualifyingEx to set
+		 */
+		public void setQualifyingEx(String qualifyingEx) {
+			this.qualifyingEx = qualifyingEx;
+		}
+		/**
+		 * @return the valueInput
+		 */
+		public String getValueInput() {
+			return valueInput;
+		}
+		/**
+		 * @param valueInput the valueInput to set
+		 */
+		public void setValueInput(String valueInput) {
+			this.valueInput = valueInput;
+		}
+		/**
+		 * @return the verb
+		 */
+		public String getVerb() {
+			return verb;
+		}
+		/**
+		 * @param verb the verb to set
+		 */
+		public void setVerb(String verb) {
+			this.verb = verb;
+		}
+		    
+	}
+	
+	public static class BpPropertyTemplateNoConditionEn {
```

Comment:
What is the difference between this and the BpPropertyTemplateNoConditionDe? Why not use one Class twice???

Reply:
Or create some inheritance structure.